kytos/utils/napps.py

```diff
@@ -332,7 +332,7 @@ def _check_module(folder):
             folder (pathlib.Path): Module path.
         """
         if not folder.exists():
-            folder.mkdir()
+            folder.mkdir(parents=True, exist_ok=True)
```

Comment:
@renanrodrigo, please add here (on `mkdir`) another argument: `mode=0o755` (the default is `777` if I am not wrong).